### PR TITLE
adds dark and light theme colors for Safari 15 

### PR DESCRIPTION
**Pull request by sarthaktexas**

### CODE REVIEW

**Comment by sarthaktexas** [comment]
y'all what these stupid tests i give up

File: packages/meta/README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Example for [hackathons.hackclub.com](https://hackathons.hackclub.com):
   description="List of upcoming high school hackathons" // page description
   image="https://hackathons.hackclub.com/card.png" // large summary image URL
   color="#ec3750" // theme color
+  darkcolor="#17171d" // theme color but for dark mode
```
Comment (sampoder) [inline packages/meta/README.md:27]:
```suggestion
  darkColor="#17171d" // theme color but for dark mode
```

File: packages/meta/src/index.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const Meta = ({
   description,
   image,
   color = '#ec3750',
+  darkcolor = '#17171d',
```
Comment (sampoder) [inline packages/meta/src/index.js:13]:
```suggestion
  darkColor = '#17171d',
```

File: packages/meta/src/index.js [hunk 3]
```diff
@@ -39,7 +40,8 @@ const Meta = ({
         <meta key="tw_img" name="twitter:image" content={image} />
       </React.Fragment>
     )}
-    <meta key="theme_color" name="theme-color" content={color} />
+    <meta key="theme_color" name="theme-color" content={color} media="(prefers-color-scheme: light)" />
+    <meta key="theme_color" name="theme-color" content={darkcolor} media="(prefers-color-scheme: dark)" />
```
Comment (sampoder) [inline packages/meta/src/index.js:44]:
```suggestion
    <meta key="theme_color" name="theme-color" content={darkColor} media="(prefers-color-scheme: dark)" />
```

**Pull request closed without merging**